drivers/flash/flash_stm32_qspi.c

```diff
@@ -1177,6 +1203,11 @@ static int flash_stm32_qspi_init(const struct device *dev)
 	/* Run IRQ init */
 	dev_cfg->irq_config(dev);
 
+#if STM32_QSPI_RESET_CMD
+	flash_stm32_qspi_send_reset(dev);
+	k_busy_wait(DT_INST_PROP(0, reset_cmd_wait));
+#endif
```

Comment:
Waiting loop inside the flash_stm32_qspi_send_reset function
Then 
  ` if (IS_ENABLED(DT_INST_PROP(0, reset_cmd))`

Reply:
That's fine to use `STM32_QSPI_RESET_CMD`.